apps/site-nft/src/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta name="author" content="">
+  <meta name="description" content="">
+  <!-- The first thing in any HTML file should be the charset -->
+  <meta charset="utf-8">
+
+  <!-- Make the page mobile compatible -->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <!-- Allow installing the app to the homescreen -->
+  <meta name="mobile-web-app-capable" content="yes">
+
+  <link rel="icon" href="/favicon.ico" />
+  <title>TheCoin NFT - Growing a better future</title>
```

Comment:
Non translatable text 😭

Reply:
OOOoohhhhh!!!!

Good catch!  This is on all our sites... how do we do the translation for an element outside react?

Will log a bug!  May require some investigation

Reply:
https://github.com/thecointech/thecoin/issues/214

Reply:
https://github.com/thecointech/thecoin/issues/216